dist/profile/templates/jenkinscontroller/casc/clouds-ec2.yaml.erb

```
@@ -156,6 +157,9 @@ jenkins:
                   }
                 }
 
+                <%- if $customDataDiskNotUsedForDocker -%>
```

```suggestion
                <%- if agent['customDataDiskNotUsedForDocker'] -%>
```

Using ruby + puppet "checks": either the key exists and has the value `true` or it does not

---

My condition was more complex in the beginning, hence this var.

As it's simpler now, your suggestion make sense, PR updated.